old_version/assets/js/main.js

```diff
@@ -296,15 +298,17 @@ inputRed.addEventListener("click", handleColorTheme);
 inputGrey.addEventListener("click", handleColorTheme);
 inputGum.addEventListener("click", handleColorTheme);
 inputPurple.addEventListener("click", handleColorTheme);
+//END OF EMMA
 
+//START OF INMA
 //LINKING FONT FAMILY TO USER CARD
 
 const ubuntuFont = document.getElementById("font-ubuntu");
 const comicFont = document.getElementById("font-comic");
 const montseFont = document.getElementById("font-montse");
 const fontCard = document.querySelector(".card-header");
 
-function handleFonttheme() {
+function handleFonttheme(event) {
   fontCard.classList.remove("font-ubuntu", "font-comic", "font-montse");
 
   const fontSelectedByUser = event.currentTarget;
```

Comment:
maybe we can use ternary here:

```
fontSelectedByUser === ubuntuFont ? fontCard.classList.add("font-ubuntu"):
fontSelectedByUser === comicFont ? fontCard.classList.add("font-comic"):
fontSelectedByUser === montseFont ? fontCard.classList.add("font-montse"):console.log("undefined");
```
**Attention!**When refreshing typography is not updated, although if saved in localStorage. Does the same thing happen to you? Maybe we need to review it.